### ISSUE TITLE
About architectures and hyper-parameter settings

### ISSUE THREAD

**Issue by dongqian0206** [kaiyuanmifen/GFNDropout#1]
Hi Dianbo,

Thank you for making the codes available. 

I am working on reproducing the results mentioned in the paper. However, based on your released codes and the method listed in your paper, I got much inferior results on the MNIST dataset, especially for the rotated images. 

Could you please provide details about network architectures, including likelihood: p(y | x, z), posterior q(z | x, y) and prior p(z | x), as well as hyper-parameter settings, including learning rate and beta?

Looking forward to hearing from you.

Best,

Dong
